src/components/CustomerModal/index.js

```diff
@@ -36,12 +36,10 @@ const CustomerModal = ({
 	onDismiss,
 	onValidate,
 	noSelect,
-	customer,
+	customer = {},
 }) => {
 	const {data, error} = useQuery(GET_ALL_CUSTOMERS, {skip: noSelect});
 
-	customer = customer || {};
-
```

Comment:
Let's be careful with this, default parameter don't work with null. https://medium.com/@bmb21/default-parameters-and-nulls-in-javascript-9149c8d26b0a

Reply:
thanks I thought it was here for something. I will just rename the result.